.github/workflows/oam-tests.yml

```
@@ -73,7 +74,12 @@ jobs:
           username: ${{env.GMAIL_TO_AUTH}}
           password: ${{env.GMAIL_PASSWORD_TO_AUTH}}
           server_port: 465
-          subject: It's OAM tests. We've failed!
-          to: ${{env.GMAIL_TO_AUTH}}
-          from: OAM Hero <botuser-T029PK78X-B085C3ZUC5N@slack-bots.com>
-          body: OAM tests job's failed. Please check oam-monitoring channel in Kontur Slack for details. Also check workflow runs page for even more details.
+          subject: We're OAM tests and we have failed!
+          to: ${{env.EMAILS_TO_NOTIFY}}
+          body: |
+            OAM tests job's failed. 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)